### DISCUSSION TITLE
Revivie + Objective + Shield = Issue?

### DISCUSSION THREAD

**Discussion by u1ps** [etlegacy/etlegacy#1938] [Ideas]
When you get revived on top of obective (documents). You will take the objectives and be shielded. This has happened several times in game and feels like a exploit as it gives big advantage on holding objective. Defending team is helpless during that shield time.

You can revive teammate on objective and use the immortality time to suppress enemy team. I dont know if this is normal behavior but being able to pick up objectives during shield feels like an exploit to me?

In a very competitive game this could give valuable few seconds.

**Comment by mittermichal** [comment]
its been always part of the game and I never seen someone complain about it
